packages/cip30/src/Wallet/types.ts

```diff
@@ -31,14 +42,15 @@ export type GetBalance = () => Promise<Cbor>;
  *
  * The results can be further paginated by `paginate` if it is not `undefined`.
  *
- * Errors: `ApiError` | `PaginateError`
+ * @throws ApiError
+ * @throws PaginateError
```

Comment:
The [spec states](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0030#apigetusedaddressespaginate-paginate--undefined-promisecboraddress) that `GetUsedAddresses` doesn't throw `PaginateError`, only `GetUtxos` does. But it is probably wrong and we should correct it.